### ISSUE TITLE
md17 dataset support

### ISSUE THREAD

**Issue by xufliu** [pyg-team/pytorch_geometric#9898]
### 🚀 The feature, motivation and pitch

I am currently using SchNet and DimeNet++. These two methods perform well on the QM9 dataset. However, when I try to train them on the MD17 dataset, I encounter two issues:

1. There is no way to calculate the gradient from Energy to Pos, making it impossible to compute Force, which is a crucial feature.

2. The results obtained by training with only the L1 loss on energy are suboptimal, even after performing a grid search. Typically, the training process should include the L1 loss on Force as well.

Could you please provide a script for training SchNet or DimeNet++ on the MD17 dataset or share pre-trained weights?
